config/jobs/kubernetes/publishing-bot/publishing-bot-presubmits.yaml

```diff
@@ -47,6 +47,30 @@ presubmits:
     annotations:
       testgrid-dashboards: sig-release-publishing-bot
       testgrid-tab-name: validate-rules
+  - name: pull-publishing-bot-test-kubernetes-master
+    always_run: false
```

Comment:
Since this would take very long, maybe we should keep this as optional everytime so it isn't triggered for each PR on every small change. @dims thoughts?

If we do this, let's also document it here with a comment.

Reply:
agree on all points @nikhita

Reply:
> Since this would take very long, maybe we should keep this as optional everytime so it isn't triggered for each PR on every small change. @dims thoughts?

> If we do this, let's also document it here with a comment.

From https://github.com/kubernetes/publishing-bot/pull/334#issuecomment-1496723061, looks like it should only take 25-30 mins. Let's avoid documenting this for now. We'll keep it as option now and reconsider depending on how the job works.